jekyll/_cci2/faq.md

```diff
@@ -248,6 +250,66 @@ Yes, the billing is associated with the organization. You can buy while within t
 {:.no_toc}
 A container is a 2 CPU 4GB RAM machine that you pay for access to. Containers may be used for concurrent tasks (for example, running five different jobs) or for parallelism (for example, splitting one job across five different tasks, all running at the same time). Both examples would use five containers.
 
+## Billing - Credit Usage Based
+
+The following FAQ questions are for our __Credit Usage Plans_.
+
+### How do the new pricing plans affect me as a customer?
+{:.no_toc}
+For the vast majority of customers, you can keep your current plan for now and this simply represents a new option you may want to consider. 
+
+### What are credits?
+{:.no_toc}
+Credits are used to pay for your usage based on machine type and size.
+
+For example, the 25,000 credit package would provide 2,500 build minutes when
+using a single machine at the default rate of 10 credits per minute. The same package would last 1,250 minutes when using 2x parallelism or 250 minutes at 10x parallelism.
+
+### Can I get one workflow run to use 50 containers at once without queuing?
+{:.no_toc}
+The Performance plan will experience less queuing, so you may run 50x parallelism builds and kick off a bunch concurrently, but this will increase your credit usage significantly.  The Seed plan has a maximum of 4x parallelism.
+
+### If a container is used for under one minute, do I have to pay for a full minute? 
+{:.no_toc}
+You pay to the nearest credit. First we round up to the nearest second, and then up to the nearest credit. On the 2x4 (medium) box this is effectively in 6 second increments. 
+
+### How do I buy credits? Can I buy in any increments?
+{:.no_toc}
+Every month, you are charged for your selected credit package at the beginning of the month.
+
+### What happens when I run out of credits?
+{:.no_toc}
+On the Performance plan, your credit package will be refilled at 10% of your monthly selected package size when you have 5% remaining on your credit package so builds will never be blocked on a lack of credits in your account.
```

Comment:
We don't refill _at_ 10%, we refill _exactly_ 10% of your monthly selection package size when you reach 5% remaining. If your monthly package size is 25,000 credits, you will automatically be refilled 2,500 credits when you reach 1,250 remaining credits.